src/components/Search/index.jsx

```diff
@@ -44,44 +28,45 @@ export default function Search({ indices }) {
       ),
     []
   );
-  const theme = useStyles();
+  const classes = useStyles();
   const [open, setOpen] = React.useState(false);
-  const handleClick = (event) => {
+  const handleInput = (event) => {
+    setAnchorEl(event.currentTarget);
     setOpen(true);
   };
   const handleClose = () => {
     setOpen(false);
   };
   return (
-    <ThemeProvider theme={theme.root}>
-      <Box mb={2}>
-        <ClickAwayListener onClickAway={handleClose}>
-          <InstantSearch
-            searchClient={searchClient}
-            indexName={indices[0].name}
-            onSearchStateChange={({ query }) => setQuery(query)}
-          >
-            <SearchBox onInput={handleClick} />
-            {open ? (
-              <Box position={'relative'} zIndex={2} mr={1}>
-                <List
-                  className={theme.popover}
-                  disableAutoFocus={true}
-                  disableEnforceFocus={true}
-                >
-                  <Box border={1} className={theme.resultsBorder}>
-                    <SearchResult
-                      className={theme.search}
-                      show={query && query.length > 0 && open}
-                      indices={indices}
-                    />
-                  </Box>
-                </List>
-              </Box>
-            ) : null}
-          </InstantSearch>
-        </ClickAwayListener>
-      </Box>
-    </ThemeProvider>
+    <Box mb={4}>
+      <InstantSearch
+        searchClient={searchClient}
+        indexName={indices[0].name}
+        onSearchStateChange={({ query }) => setQuery(query)}
+      >
+        <SearchBox onInput={handleInput} />
+        <Popover
+          open={open}
+          anchorEl={anchorEl}
+          onClose={handleClose}
+          anchorOrigin={{
+            vertical: 'bottom',
+            horizontal: 'left',
+          }}
+          disableAutoFocus
+          disableEnforceFocus
+          classes={{ paper: classes.popover }}
+        >
```

Comment:
Here's the material-ui `Popover` implementation! `anhorEl` refers to the search box, so it can position itself left bottom of the search box.